In-house patch to remove the dependency on unittest2. Python 2.7
includes unittest, which is equivalent to unittest2 for Python 2.6. If
this line is present, stevedore will fail to find the pip package for
it, which causes a failure.

--- testtools-2.0.0/requirements.txt.~1~	2016-02-03 04:04:01.000000000 +0000
+++ testtools-2.0.0/requirements.txt	2016-09-06 16:55:05.201984120 +0000
@@ -5,5 +5,4 @@ pyrsistent
 # 'mimeparse' has not been uploaded by the maintainer with Python3 compat
 # but someone kindly uploaded a fixed version as 'python-mimeparse'.
 python-mimeparse
-unittest2>=1.0.0
 traceback2
--- testtools-2.0.0/testtools.egg-info/requires.txt.~1~	2016-02-04 02:45:21.000000000 +0000
+++ testtools-2.0.0/testtools.egg-info/requires.txt	2016-09-06 16:55:18.120421746 +0000
@@ -3,9 +3,4 @@ extras
 fixtures>=1.3.0
 pyrsistent
 python-mimeparse
-unittest2>=1.0.0
 traceback2
-
-[test]
-testscenarios
-unittest2>=1.1.0
